src/Microsoft.VisualBasic.Core/tests/ConversionTests.cs

```diff
@@ -207,43 +212,94 @@ public void Fix(decimal value, decimal expected)
             Assert.Equal(expected, Conversion.Fix(value));
         }
 
+        [Theory]
+        [InlineData(char.MinValue)]
+        [InlineData(char.MaxValue)]
+        [MemberData(nameof(Various_ArgumentException_TestData))]
+        public void Fix_ArgumentException(object value)
+        {
+            Assert.Throws<ArgumentException>(() => Conversion.Fix(value));
+        }
+
+        [Theory]
+        [InlineData(null)]
+        public void Fix_ArgumentNullException(object value)
+        {
+            Assert.Throws<ArgumentNullException>(() => Conversion.Fix(value));
+        }
+
         public static IEnumerable<object[]> Fix_Short_TestData()
         {
+            yield return new object[] { short.MinValue, short.MinValue };
+            yield return new object[] { (short)-1, (short)-1 };
             yield return new object[] { (short)0, (short)0 };
+            yield return new object[] { (short)1, (short)1 };
+            yield return new object[] { short.MaxValue, short.MaxValue };
             // Add more...
```

Comment:
>// Add more... [](start = 12, length = 14)

Feel free to remove these comments (and the corresponding `yield break;` statements) in each of these cases.